pkgs/development/python-modules/shlib/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, braceexpand
+, inform
+}:
+
+buildPythonPackage rec {
+  pname = "shlib";
+  version = "1.5";
+
+  src = fetchFromGitHub {
+    owner = "KenKundert";
+    repo = "shlib";
+    rev = "v${version}";
+    sha256 = "sha256-2fwRxa64QXKJuhYwt9Z4BxhTeq1iwbd/IznfxPUjeSM=";
+  };
+
+  pythonImportsCheck = [ "shlib" ];
+  propagatedBuildInputs = [ braceexpand inform ];
+
```

Comment:
Looks like that upstream is running the tests with `pytest`. Please check if we can run them with `pytestCheckHook`.